src/metabase/api/public.clj

```diff
@@ -188,7 +188,7 @@
 (api/defendpoint GET "/card/:uuid/query/:export-format"
   "Fetch a publicly-accessible Card and return query results in the specified format. Does not require auth
   credentials. Public sharing must be enabled."
-  [uuid export-format :as {{:keys [parameters format_rows]} :params}]
+  [uuid export-format :as {{:keys [parameters format_rows pivot_results]} :params}]
```

Comment:
since this endpoint is still a GET, pass both format_rows and pivot_results as query params here.